### PR TITLE
chore(.github/workflows): Bump lint version

### PR DESCRIPTION
**Pull request by walldiss**

### CODE REVIEW

**Review by Wondertan — COMMENTED** [review]
Seems like upgrading requires fixing all the new linter rules

File: Makefile [hunk 1]
```diff
@@ -88,7 +88,7 @@ fmt: sort-imports
 ## lint: Linting *.go files using golangci-lint. Look for .golangci.yml for the list of linters.
 lint: lint-imports
 	@echo "--> Running linter"
-	@golangci-lint run
+	@golangci-lint rungolangci-lint run
```
Comment (Wondertan) [inline Makefile:91]: typo?

**Review by Wondertan — COMMENTED** [review]
Thank you for this!

File: Makefile (hunk 1 above)
Comment (walldiss) [reply Makefile:91]: yep

**Review by Wondertan — DISMISSED** [review]

**Review by distractedm1nd — APPROVED** [review]

**Review by derrandz — DISMISSED** [review]
ran locally, works like MUAH

**Review by distractedm1nd — APPROVED** [review]

**Review by Wondertan — APPROVED** [review]

**Pull request merged**